/***********************************************************************
 * This file is part of Scanned Image Extract.
 *
 * Scanned Image Extract is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 * 
 * Scanned Image Extract is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 * 
 * You should have received a copy of the GNU General Public License
 * along with Scanned Image Extract.  If not, see <http://www.gnu.org/licenses/>
 * 
 * 
 * Copyright (C) 2015, Dominik Rue√ü; info@dominik-ruess.de
 **********************************************************************/

#ifndef DIALOGABOUT_SCANNER_EXTRACT_H
#define DIALOGABOUT_SCANNER_EXTRACT_H

#include <QDialog>

#include "about.h"

namespace Ui {
class DialogAbout;
}

class DialogAbout : public QDialog
{
    Q_OBJECT
    
public:
    explicit DialogAbout(QWidget *parent = 0);
    ~DialogAbout();

    void setVersion(const int, const int, const int, const int);

    QString getDonateText() const;
    
private slots:
    void on_pushButton_done_clicked();

private:
    Ui::DialogAbout *ui;
};

#endif // DIALOGABOUT_SCANNER_EXTRACT_H
